mappings/net/minecraft/network/packet/s2c/play/ChatMessageS2CPacket.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_7439 net/minecraft/network/packet/s2c/play/ChatMessageS2CPacket
```

Comment:
Should we name it `UnsignedChatMessageS2CPacket` now that we have a signed version?

Reply:
hm, we could name it like this, not sure. I kept the name from the older versions.

Reply:
@Bixilon The old name is "GameMessage" and I think that one is better (I don't think people call things like death messages "chat message")

Reply:
If we go for `GameMessage`, `Signed` prefix could be dropped. (Iirc, an unsigned message can still be sent with this packet, need to check)

Reply:
Yes, but it appeares in the chat (except for hotbar, but that is kind of a special type?)